src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -39,6 +39,8 @@ abstract class AbstractObjectNormalizer extends AbstractNormalizer
     const MAX_DEPTH_HANDLER = 'max_depth_handler';
     const EXCLUDE_FROM_CACHE_KEY = 'exclude_from_cache_key';
 
+    public const ARRAY_OBJECTS_KEY = 'array_objects';
```

Comment:
these constants usually have no "key" suffix. the one above is talking about the "cache key". i'd call this something like `PRESERVE_EMPTY_OBJECTS`